pandas/core/indexes/datetimes.py

```diff
@@ -1922,6 +1925,39 @@ def tz_convert(self, tz):
         ------
         TypeError
             If DatetimeIndex is tz-naive.
+
+        See Also
+        --------
+        tz_localize : Localize tz-naive DatetimeIndex to given time zone,
+                      or remove timezone from tz-aware DatetimeIndex.
+
+        Examples
+        --------
+        >>> dti = pd.DatetimeIndex(start='2014-08-01 09:00',
+        ...                       freq='H', periods=3, tz='Europe/Berlin')
+
+        >>> dti
+        DatetimeIndex(['2014-08-01 09:00:00+02:00',
+                       '2014-08-01 10:00:00+02:00',
+                       '2014-08-01 11:00:00+02:00'],
+                        dtype='datetime64[ns, Europe/Berlin]', freq='H')
+
+        With the `tz` parameter, we can change the DatetimeIndex
+        to other time zones:
+
+        >>> dti.tz_convert('US/Eastern')
```

Comment:
so THIS doesn't work (well it works but doesn't do what you want).

Create 2 example dti's here, the first with NO timezone (like you had originally), and ``.tz_localize('Europe/Berlin')``
the second with tz='Europe/Berlin'  then use ``.tz_localize(None)``, this shows removing it.

Reply:
In case of tz_convert(tz), do we have to use tz_localize() ?